### ISSUE TITLE
Printing selected map areas 

### ISSUE THREAD

**Issue by dpalomino** [Cadasta/cadasta-platform#915]
Product backlog id: 45.00

"As a user, I want to print a selected area of a map including the location geometries contained in it."

The idea would be to generate a pdf document containing the area of the map that is currently displayed on GUI, and the location geometries in that area. In the header we should include the organization and the project name, and also include some blank space for notes. 

This will be very helpful for the verification phase on the field. So once that all data and geometries have been collected, to be able to print a map with that info and go back to the community for their verification. 

Certainly a plus would be to have the possibility of printing this map in different sizes (A4, A3). 

@ian-ross, I think you are working on something similar for Field Papers, isn't it? Do you think there is something we can reuse of if this is simply going to be covered by Field Papers feature?

**Comment by ian-ross** [comment]
I don't think the Field Papers stuff would be usable for this as is.  It's designed to generate multi-page atlases to cover a region, and does that using relatively high-resolution map data, meaning that the atlas generation needs to run as an asynchronous background process.  It's not really suitable for the kind of quick print generation you seem to be talking about here.  It's probably also worth saying that whether this kind of quick print production is feasible depends on just how we deal with location data rendering: if you have a project with 50,000 polygons and hit the print button when the view is zoomed right out, how do you ensure that the print production happens quickly?

**Comment by wonderchook** [comment]
One thing to think about is what the use of the map specifically is. @dpalomino you mention for verification. I could see a single map maybe being useful for verification of a single community boundary, but not very useful for verification of individual locations in a slum for example. I know some people previously were building on top of the idea of Field Papers to do verification. In that scenario they were printing the data on a series of atlases. This might not be what we want to do, but I think it makes sense to explore a bit further the end use of the map.

**Comment by dpalomino** [comment]
Yes, I was thinking on verification of community boundaries, or small areas of a village with a few parcels for instance. Something that could fit in a single A4 (or maybe A3). For bigger areas I think a multi-page atlas with Field Papers would be more suitable. 

Maybe we could limit the number of spatial units contained in the area to print (perhaps 50 or 100?).
